<launch>
        <node pkg="roiViewer" type="roiViewer_node" name="roiViewer" output="screen">
                <!-- mode = {roi_display} -->
                <param name="mode" type="string" value="roi_display"/> -->
		<!-- Note: label should be updated to list of values -->
                <param name="label" type="int" value="1" />
		<remap from ="input_image" to="camera/rgb/image_color"/>
		<remap from ="input_rois" to="output_rois1"/>  
        </node>
</launch>

